dbus-opendtu.py

```diff
@@ -119,8 +119,9 @@ def _readConfigDTU(self, actual_inverter):
     try: 
       self.acposition         = int(config['INVERTER{}'.format(self.pvinverternumber)]['AcPosition'])
     except:
-      self.acposition         = int(config['DEFAULT']['AcPosition'])
-      logging.error("Deprecated AcPosition DEFAULT entries must be moved to INVERTER section")
+      logging.error("Deprecated Host ONPREMISE entries must be moved to DEFAULT section")
+      raise ValueError("Deprecated Host ONPREMISE entries must be moved to DEFAULT section")
+      exit
```

Comment:
Nach dem raise ist das exit wirkungslos, weil die Exception eh dazu führt, dass er dort abbricht und beim "nächsten" (in der Call-Hierarchier) catch weiter macht

Reply:
Sorry, ich habe vorhin "exit" geschrieben, wo eigentlich raise richtig ist.

Reply:
exit ist überall raus